packages/grid/src/Grid.tsx

```diff
@@ -1989,27 +1989,32 @@ class Grid extends PureComponent<GridProps, GridState> {
 
   render(): ReactNode {
     const { cursor } = this.state;
+    const { children } = this.props;
 
     return (
-      <>
+      // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+      <div
+        style={{ height: '100%', width: '100%' }}
```

Comment:
Actually this can be removed now b/c I'm no longer rendering anything as a Grid child

But no, it wouldn't need position relative since there's no relative shift like top or left props